R/pin-read-write.R

```diff
@@ -111,14 +111,15 @@ guess_type <- function(x) {
 }
 
 object_write <- function(x, path, type = "rds") {
-  type <- arg_match0(type, c("rds", "json", "arrow", "pickle", "csv"))
+  type <- arg_match0(type, c("rds", "json", "arrow", "pickle", "csv", "qs"))
```

Comment:
Also need to update docs above; and might make sense to change this to `setdiff(object_types, "file")`